drivers/system_pathways/app/views/system_pathways/warehouse_reports/reports/_pathways_chart.haml

```diff
@@ -7,6 +7,15 @@
 %h4 #{pluralize(number_with_delimiter(@pathways_chart.node_clients('Served by Homeless System').count), 'Total Client')} Served by The Homeless System
 .sankey-chart.mb-6{style: 'position: relative;'}
   #sankey
+%p The above chart tracks clients with open homeless enrollments in the selected universe.  System pathways are limited to those that proceed through the continuum.  If a client goes backwards within the continuum (from more housed to less housed) the regression is ignored for the puposes of this report unless they first exited to a permanent destination.  Clients who exited to a permanent destination and then subsequently enrolled in a homeless project are counted in the Returns to Homelessness category.  The progression of least to most homeless is as follows:
+%ul
+  %li SO (4)
+  %li ES (1)
+  %li SH (8)
+  %li TH (2)
+  %li PH - RRH (13)
+  %li PH - PSH (3), PH - PH (9), PH - OPH (10)
+
```

Comment:
I don't understand what 'least to most homeless` means, and how it relates to the list...